target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -359,6 +363,11 @@ ath79_setup_macs()
 	devolo,magic-2-wifi)
 		label_mac=$(macaddr_add "$(mtd_get_mac_binary art 0x1002)" 3)
 		;;
+	dlink,dap-1330-a1|\
+	dlink,dap-1365-a1)
+		lan_mac=$(mtd_get_mac_text "mp" 0x1)
+		label_mac=$lan_mac
+		;;
```

Comment:
@ynezz If you merge both dlink PRs from this author, I think this node can be merged for them.